restservice/src/test/java/org/itevents/service/geocoder/YandexGeocoderServiceTest.java

```diff
@@ -0,0 +1,52 @@
+package org.itevents.service.geocoder;
+
+import com.google.gson.Gson;
+import com.yandex.geocoder.api.geocoder.Geocoder;
+import com.yandex.geocoder.api.geocoding.GeocoderRequest;
+import com.yandex.geocoder.api.geocoding.GeocoderResponse;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import javax.inject.Inject;
+import java.io.IOException;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+/**
+ * Created by zavodnyuk on 09.04.2016.
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(locations = { "classpath*:applicationContext.xml" })
+public class YandexGeocoderServiceTest {
+
+    private String ADDRESS = "Киев, площадь Независимости, 1";
+
+    @Inject
+    @InjectMocks
+    private YandexGeocoderService addressLocationService;
+
+    @Inject
+    @Mock
+    private Geocoder geocoder;
+
+    @Before
+    public void init() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void shouldGeocode() throws IOException {
```

Comment:
@zavodnyuk you need to assert `Location` object that is returned when `addressLocationService.getLocationByAddress(...)` is invoked